xcode/Subconscious/Shared/Components/Detail/MemoEditorDetail.swift

```diff
@@ -1187,10 +1204,47 @@ struct MemoEditorDetailModel: ModelProtocol {
         state: MemoEditorDetailModel,
         environment: AppEnvironment
     ) -> Update<MemoEditorDetailModel> {
-        let pollFx = AppEnvironment.poll(every: Config.default.pollingInterval)
-            .map({ _ in MemoEditorDetailAction.autosave })
```

Comment:
This task would outlive the view that created it. This _may_ point to an underlying issue in Observable store as I would expect this `fx` to be cancelled.